sql/parser/Makefile

```diff
@@ -0,0 +1,3 @@
+sql.go: sql.y
+	go tool yacc -o sql.go sql.y
+	@echo "$$(echo '// Code generated by go yacc.' | cat - sql.go)" > sql.go
```

Comment:
Not easily. `go tool yacc -o -` writes to a file named `"-"`. I can do `go tool yacc -o /dev/stdout`, but that feels clunky. I'd prefer to keep the current code. This isn't called often, and what is there works.